### ISSUE TITLE
Project maintenance query

### ISSUE THREAD

**Issue by SeanFarrow** [Microsoft/AspNetCoreInjection.TypedFactories#3]
Is this project still being mained and upgraded.

I can definitely see the value this has and would like to take it forward if not. Alternatively, I'm happy to upgrade this to newer NuGet package versions and .Net versions if this would be accepted.

Thanks,
Sean.

**Comment by Xriuk** [comment]
Agree, this project works great, it needs just few minor improvements

**Comment by MisinformedDNA** [comment]
Beuller?
